docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,48 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+The `readOnly` prop set to true makes the element not mutable, meaning the user can not edit the control.
+
+```jsx
+<ReferenceArrayInput
+    source="tags_ids"
+    reference="tags"
+    readOnly
+/>
+```
+
+**Warning**: The `readOnly` prop refers to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you pass a child component, you'll have to specify the `readOnly` prop on each input:
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** `ReferenceArrayInput‏` accepts at most 1 child I believe

```suggestion
**Warning**: The `readOnly` prop is forwarded to the default [`<AutocompleteArrayInput>`](./AutocompleteArrayInput.md) child. If you pass a custom child component, you'll have to specify the `readOnly` prop on it instead:
```